### PR TITLE
Change port for docker entrypoint and margin for slice description in charts for the dashboard

### PR DESCRIPTION
**Pull request by ghost**
- Changes in chart description styles for dashboards
- Update port in entrypoint

### CODE REVIEW

File: superset/assets/stylesheets/superset.less [hunk 1]
```diff
@@ -51,7 +51,8 @@ input.form-control {
 
 .slice_description{
     padding: 8px;
-    margin: 5px;
+    margin-top: 5px;
```
Comment (hughhhh) [inline superset/assets/stylesheets/superset.less]: `margin: 5px 0` will do the same

**Comment by mistercrunch** [comment]
Can we see the before/after on the margins?

File: superset/assets/stylesheets/superset.less (hunk 1 above)
Comment (BrayanArrieta) [reply superset/assets/stylesheets/superset.less]: Thanks for the recomendation, I forgot that I could like that too.

**Comment by BrayanArrieta** [comment]
@mistercrunch yes of course, the change it's in the view but I want to apply the recomendation of @hughhhh, In some time I will add some images for apreciate the change.

**Comment by BrayanArrieta** [comment]
# Previous margin 5px to all sides
image

# New possible change margin just for top and bottom
image

File: contrib/docker/docker-entrypoint.sh [hunk 2]
```diff
@@ -4,7 +4,7 @@ set -ex
 if [ "$#" -ne 0 ]; then
     exec "$@"
 elif [ "$SUPERSET_ENV" = "local" ]; then
-    flask run -p 8080 --with-threads --reload --debugger  --host=0.0.0.0
+    flask run -p 8088 --with-threads --reload --debugger  --host=0.0.0.0
```
Comment (hughhhh) [inline contrib/docker/docker-entrypoint.sh]: If you change the port number make sure to update the docs that are referencing this

**Comment by kristw** [comment]
Please update the PR title to be a bit more specific.

**Comment by phoenix24** [comment]
I guess, this PR is superseded by [6504](https://github.com/apache/incubator-superset/pull/6504) with respect the changes to `docker-entrypoint.sh`.

**Comment by BrayanArrieta** [comment]
@phoenix24 I think that you have the reason about PR is superseded in de `docker-entrypoint.sh`. I will close this PR and add just another for the changes in views thanks !.

**Comment by Marcofeli** [comment]
@BrayanArrieta

**Pull request closed without merging**